src/backend/controllers/userController.js

```diff
@@ -161,8 +161,11 @@ exports.login = async (req: Request, res: Response) => {
 		// See if User exists - might change this to const
 		const user = await User.findOne({ userName: userName });
 
+		// Internal password constant
+		const passCheck = (user) ? user.password : "";
```

Comment:
sketchy null check. `user!=null` preferred if you are trying to do this. Also single quotes are preferred in JS unless the string contains a single quote.

Even better is:
`const passCheck = user?.password ?? '';`